### PR TITLE
Solo CTA widget updates & fixes

### PR DESCRIPTION
**Pull request by karloscastaneda**

### CODE REVIEW

File: static/solo-cta/index.js [hunk 1]
```diff
@@ -280,21 +279,52 @@ function handleModalInitialized(e){
...
-
-  if (utils.isIOS) {
-    var onOrientationChange = function () {
-        if (iframeModal && iframeModal.contentWindow) {
-            var width = iframeModal.parentNode.offsetWidth;
-            iframeModal.contentWindow.window.postMessage({
-                event: config.eventPrefix + ':modal_holder_resize',
-                size: [width, Math.floor(width * (9 / 16))]
-            },'*');
+  var detectOrientation = function(){
```
Comment (rfornes) [inline static/solo-cta/index.js]: I really need to understand what issues are being tackled here, thanks.
Ticket will work otherwise please explain.

Comment (rfornes) [reply static/solo-cta/index.js]: Sounds related?
https://tvpage.atlassian.net/browse/DEV-7876

Comment (karloscastaneda) [reply static/solo-cta/index.js]: @luisvelazquex can you help here, i think this was you right?

Comment (rfornes) [reply static/solo-cta/index.js]: @luisvelazquex I think we need to revisit this fix. Please remove this code from the branch.
There's a potential performance issue in the code, I explain this below.

File: static/solo-cta/index.js [hunk 2]
```diff
@@ -280,21 +279,52 @@ function handleModalInitialized(e){
...
+              iframeModal.contentWindow.window.postMessage({
+                  event: config.eventPrefix + ':modal_holder_resize',
+                  size: [width, Math.floor(width * (9 / 16))]
+              },'*');
+          }
+        },500);
     };
     var orientationChangeEvent = 'onorientationchange' in window ? 'orientationchange' : 'resize';
     window.removeEventListener(orientationChangeEvent,onOrientationChange, false);
     window.addEventListener(orientationChangeEvent,onOrientationChange, false);
```
Comment (rfornes) [inline static/solo-cta/index.js]: The function `onOrientationChange` is going to be called every time there's an orientation change.
The function uses a `setInterval` function that will run each half of a second and it won't be stopped at any time.
Take a look at https://jsfiddle.net/5oxLrqg5/1/

Please remove code, we need to figure a better way to fix this issue

**Pull request closed without merging**